### DISCUSSION TITLE
compilation errors in Unity

### DISCUSSION THREAD

**Discussion by chenruijia120** [MixedRealityToolkit/MixedRealityToolkit-Unity#533] [Q&A]
Hi! I got similar errors with https://learn.microsoft.com/en-us/answers/questions/1392297/i-am-getting-compilation-errors-in-unity-that-nonn. Library\PackageCache\org.mixedrealitytoolkit.uxcore@45c07beda63c\Experimental\NonNativeKeyboard\TMPInputFieldNonNativeKeyboardTrigger.cs(67,42): error CS0103: The name 'NonNativeKeyboard' does not exist in the current context

I am using Unity 2023.1.17f1 and Windows 11 22H2. I followed the instructions here: https://learn.microsoft.com/en-us/training/modules/learn-mrtk-tutorials/1-5-exercise-configure-resources. Could you please help me?

**Comment by marlenaklein-msft** [comment]
Resolved here https://github.com/microsoft/MixedRealityLearning/issues/81
